purview/models/koji.py

```diff
@@ -9,6 +9,7 @@ class KojiBuild(StructuredNode):
     id_ = UniqueIdProperty(db_property='id')
     tasks = RelationshipTo('KojiTask', 'TRIGGERS')
     owner = RelationshipTo('.user.User', 'OWNED_BY')
+    tags = RelationshipTo('KojiTag', 'TAGGED_BY')
```

Comment:
Based on our discussion, this should likely be called `CONTAINED_BY` for more generic relationship names.